vault/data_source_pki_secret_backend_issuer.go

```diff
@@ -0,0 +1,124 @@
+// Copyright (c) HashiCorp, Inc.
+// SPDX-License-Identifier: MPL-2.0
+
+package vault
+
+import (
+	"context"
+	"fmt"
+	"log"
+
+	"github.com/hashicorp/terraform-plugin-sdk/v2/diag"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+
+	"github.com/hashicorp/terraform-provider-vault/internal/consts"
+	"github.com/hashicorp/terraform-provider-vault/internal/provider"
+)
+
+func pkiSecretBackendIssuerDataSource() *schema.Resource {
```

Comment:
I know I'm asking a lot :) 

Would it be difficult to add `vault_pki_secret_backend_issuers` and `vault_pki_secret_backend_keys` data sources that would return a list of issuer/key ids. I think this would help out with migration workflows to find/match existing issuers/keys without knowing ahead of time.

https://developer.hashicorp.com/vault/api-docs/secret/pki#list-issuers
https://developer.hashicorp.com/vault/api-docs/secret/pki#list-keys

Reply:
Sure thing, thanks for the feedback! Will add those in as well